docs/how-to/deploy_sdcore_user_plane_in_dpdk_mode.md

```diff
@@ -0,0 +1,125 @@
+# Deploy SD-Core User Plane in DPDK mode
+
+This guide covers how to deploy the User Plane Function (UPF) in DPDK mode using 
+the `sdcore-user-plane` Juju bundle.
+
+## Requirements
+
+- A Kubernetes cluster which meets below requirements:
+  - host CPU that supports AVX2, RDRAND and PDPE1GB instructions (Intel Haswell, AMD Excavator 
+    or equivalent)
+  - SR-IOV interfaces for Access and Core networks
+  - At least two 1G HugePages available
+  - `driverctl` installed
+  - LoadBalancer with 1 available address for the UPF
+  - Multus CNI enabled
+- Juju >= 3.1/stable
+- A Juju controller bootstrapped onto the Kubernetes cluster
+- Juju model created
+
```

Comment:
We need to add a note that [Microk8s strictly confined](https://ubuntu.com/blog/strictly-confined-microk8s) should not be used. As we are deploying Confined MicroK8s in [getting started](https://canonical-charmed-5g.readthedocs-hosted.com/en/latest/tutorials/getting_started/), users probably continue with that version and they will fail.

Reply:
This How-To is supposed to be as K8s agnostic as possible. Once the `sriov-device-plugin` Microk8s addon is merged to Microk8s' community addons repo, I'll create another, Microk8s specific doc.